### DISCUSSION TITLE
Could I get some help in getting Galaxy Brain Badge? Thanks! 👋

### DISCUSSION THREAD

**Discussion by 0xpako** [drknzz/GitHub-Achievements#88] [Q&A] [answered]
Title ^

**Comment by mrdulin** [comment, marked as answer]
I can help.

**Reply by 0xpako** [reply]
I'll mark your answer, and will wait for a thread from you!

**Reply by mrdulin** [reply]
Here https://github.com/drknzz/GitHub-Achievements/discussions/94, let's do it.
